### PR TITLE
doc: Use plural for units for the other quantities

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

This PR is similar to PR #2251 and changes the units for other quanities to plural.
- [x] documentation
- [x] datasets
- [x] tests
- [x] examples
    - [x] gallery
    - [x] tutorials
    - [x] get started
    - [x] projections   

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Comment by yvonnefroehlich** [comment]
I am not sure about the changes in `first_figure` (please see commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/1cdd9150ace5fbd84bb9075ec044f8344e13ee36) and `rose.py` (please see commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/91135437d0b0e3b0e6396082bc4fe7e8709c35bf).

File: examples/gallery/histograms/blockm.py
Comment (seisman) [inline examples/gallery/histograms/blockm.py:19]:
```suggestion
# Define spacing in x and y direction (150x150 arc-minutes blocks)
```

Comment (yvonnefroehlich) [reply examples/gallery/histograms/blockm.py]: These changes are made in commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/28d2914d3dcf4a5f378e9517cc98dfb1392fd42e.

File: examples/gallery/histograms/blockm.py
Comment (michaelgrund) [inline examples/gallery/histograms/blockm.py:19]:
```suggestion
# Define spacing in x and y direction (150x150 arc-minute blocks)
```
In my opinion the plural is not correct here since the following word is already plural. Maybe a native speaker like @maxrjones or @willschlitzer may have a look on this.

File: examples/gallery/histograms/blockm.py
Comment (michaelgrund) [inline examples/gallery/histograms/blockm.py:25]:
```suggestion
# 150x150 arc-minute bins using blockmean
```
same as above

File: examples/tutorials/advanced/vectors.py
Comment (michaelgrund) [inline examples/tutorials/advanced/vectors.py:75]:
```suggestion
# one point is defined as 1/72 inche.
```
should be singular in my opinion, let's the what our native speakers say 😉

File: examples/gallery/histograms/blockm.py
Comment (willschlitzer) [reply examples/gallery/histograms/blockm.py:19]: I think the correct grammar is "arc-minute blocks"

File: examples/tutorials/advanced/vectors.py
Comment (willschlitzer) [reply examples/tutorials/advanced/vectors.py:75]: I think the "1/72 inch" makes more sense; the correct grammar would be "1/72 of an inch" but I think that is unnecessarily wordy.

File: examples/gallery/histograms/blockm.py
Comment (yvonnefroehlich) [reply examples/gallery/histograms/blockm.py]: Thanks @willschlitzer!
Changed it, see commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/fef0f1028561dcf8d297ec146029973a5e1a2d45.

File: examples/tutorials/advanced/vectors.py
Comment (yvonnefroehlich) [reply examples/tutorials/advanced/vectors.py]: Thanks @willschlitzer.
Changed it back to the short singular version, see commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/3fb23aecc13bb3ba98eb1bafd35e299cf85f4807.

File: examples/gallery/histograms/blockm.py
Comment (willschlitzer) [inline examples/gallery/histograms/blockm.py:45]:
```suggestion
# Calculate number of total locations within 150x150 arc-minute bins
# with blockmean's summary parameter
```

I think using "with" makes more sense than "via".

File: pygmt/src/inset.py
Comment (willschlitzer) [inline pygmt/src/inset.py:70]:
```suggestion
        a 2-character justification code *justify*.
```

File: pygmt/src/x2sys_init.py
Comment (willschlitzer) [inline pygmt/src/x2sys_init.py:91]: When stating a rate, should abbreviations be used or not? This uses both long unit names (miles/hr) and short unit names (m/s), so it should at least be standardized.

File: pygmt/src/inset.py
Comment (yvonnefroehlich) [reply pygmt/src/inset.py]: Good point @willschlitzer! Changed this in the commits https://github.com/GenericMappingTools/pygmt/pull/2278/commits/e1e402e6382e5b26042bdfe7c7c8e52381204ce6, https://github.com/GenericMappingTools/pygmt/pull/2278/commits/2d09a2a71b046670017735b8bcd0293540b69f3e, and https://github.com/GenericMappingTools/pygmt/pull/2278/commits/d4f46249f2abd4947eef400264a0723b74e32d28.

File: pygmt/src/x2sys_init.py
Comment (yvonnefroehlich) [reply pygmt/src/x2sys_init.py:91]: Yep, I saw this and changed only the first column. The second column is a bit tricky when we want to use the short cuts:
```
        - **c** - Cartesian userdist or userdist/usertime
        - **e** - meters or m/s
        - **f** - feet or feet/s                     -> feet or ft/s
        - **k** - kilometers or km/hr
        - **m** - miles or miles/hr             -> miles or mi/hr
        - **n** - nautical miles or knots        -> velocity unit is missing or not available?, short cut for nautical miles?
        - **u** - survey feet or survey feet/s    -> short cut for survey feet?
```

File: examples/gallery/histograms/blockm.py
Comment (yvonnefroehlich) [reply examples/gallery/histograms/blockm.py]: I can not say reliable what is here correct or better, but changed it :wink:.

File: pygmt/src/x2sys_init.py
Comment (yvonnefroehlich) [reply pygmt/src/x2sys_init.py:91]: Hm. How to proceed here @GenericMappingTools/pygmt-maintainers:
(i) Leave as is, with non SI units in long forms
(ii) Change all units to long forms
(iii) Change all units to short cuts, please help with the short cuts for the non SI units

Comment (willschlitzer) [reply pygmt/src/x2sys_init.py:91]: I think that using abbreviations is fine when stating rates, since the long name is already stated.

File: pygmt/src/rose.py
Comment (seisman) [inline pygmt/src/rose.py:83]: I'm not sure about this change. Google search returns "180 degree ambiguity", "180-degree ambiguity", and "180 degrees ambiguity".

**Comment by seisman** [comment]
Since we may not reach a consensus about some changes and the changes are all minor, I prefer to delay this PR to v0.9.0 instead.

File: pygmt/src/rose.py
Comment (yvonnefroehlich) [reply pygmt/src/rose.py]: I changed this back in commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/9053380919a4523010163e6331835ed46a2813f0. I also was unsure here (please see comment https://github.com/GenericMappingTools/pygmt/pull/2278#issuecomment-1365010563).

File: pygmt/src/x2sys_init.py
Comment (yvonnefroehlich) [reply pygmt/src/x2sys_init.py]: Changed the units of the velocities to their abbreviations in commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/b8f2103829bbf5e360027d9e051a601d1630f051.

**Review by seisman — APPROVED** [review]

**Comment by seisman** [comment]
Ping @GenericMappingTools/pygmt-maintainers to give this PR a final review before merging.

**Pull request merged**
